File: torrent/src/main/java/com/home/torrent/util/WLog.java
package com.home.torrent.util;

import android.annotation.SuppressLint;

import java.text.SimpleDateFormat;
import java.util.Date;

public class WLog {
    @SuppressLint("SimpleDateFormat")
    private final static SimpleDateFormat sDataFormatter = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss:sss");

    public static void i(String tag, Object info) {
        String msg = sDataFormatter.format(new Date()) + " [WindMusic] [" + Thread.currentThread() + "] [I] [" + tag + "] " + info;
        System.out.println(msg);
    }

    public static void e(String tag, Object info) {
        String msg = sDataFormatter.format(new Date()) + " [WindMusic] [" + Thread.currentThread() + "] [E] [" + tag + "] " + info;
        System.out.println(msg);
    }

    public static void d(String tag, Object info) {
        String msg = sDataFormatter.format(new Date()) + " [WindMusic] [" + Thread.currentThread() + "] [D] [" + tag + "] " + info;
        System.out.println(msg);
    }

    public static void v(String tag, Object info) {
        String msg = sDataFormatter.format(new Date()) + " [WindMusic] [" + Thread.currentThread() + "] [V] [" + tag + "] " + info;
        System.out.println(msg);
    }
}

File: torrent/src/main/java/com/home/torrent/util/WLog.kt
package com.home.torrent.util

import android.annotation.SuppressLint
import java.text.SimpleDateFormat
import java.util.Date

object WLog {
    @SuppressLint("SimpleDateFormat")
    private val sDataFormatter = SimpleDateFormat("yyyy-MM-dd hh:mm:ss:sss")
    fun i(tag: String, info: Any) {
        val msg =
            sDataFormatter.format(Date()) + " [WindMusic] [" + Thread.currentThread() + "] [I] [" + tag + "] " + info
        println(msg)
    }

    fun e(tag: String, info: Any) {
        val msg =
            sDataFormatter.format(Date()) + " [WindMusic] [" + Thread.currentThread() + "] [E] [" + tag + "] " + info
        println(msg)
    }

    fun d(tag: String, info: Any) {
        val msg =
            sDataFormatter.format(Date()) + " [WindMusic] [" + Thread.currentThread() + "] [D] [" + tag + "] " + info
        println(msg)
    }

    fun v(tag: String, info: Any) {
        val msg =
            sDataFormatter.format(Date()) + " [WindMusic] [" + Thread.currentThread() + "] [V] [" + tag + "] " + info
        println(msg)
    }
}